roles/edpm_install_certs/tasks/copy_certs_and_keys.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Copy over certs and keys
+  tags:
+    - certs
+  become: true
+  block:
+    - name: Set paths
+      ansible.builtin.set_fact:
+        cert_src_path: "/var/lib/openstack/certs/{{ service }}"
+        cert_dest_path: "/var/lib/openstack/certs/{{ service }}"
+        key_dest_path: "/var/lib/openstack/certs/{{ service }}"
+
+    - name: Set cert_dest_path for libvirt
+      ansible.builtin.set_fact:
+        cert_dest_path: "/etc/pki/libvirt"
+        key_dest_path: "/etc/pki/libvirt/private"
+      when: service  == "libvirt"
```

Comment:
im ok with this but it would likely be more scalable if you just put
the cert in /var/lib/openstack/certs/libvirt and then modify the libvirt role to copy it form there to /etc/pki/libvirt if it exits

that would allow any other service that need to be host installed to follow the same pattern without modifying this role.

Reply:
ack - will do as suggested, and add the relevant code to the libvirt role.